### PR TITLE
Line corrections

### PR DESCRIPTION
**Pull request by j0rt3g4**

### CODE REVIEW

**Comment by e0i** [comment]
@j0rt3g4 Please confirm that the PR addresses issue #3416

**Comment by j0rt3g4** [comment]
@e0i This PR Address some extra lines that were on the docs.
The issue has been solved already by the @illfated  merged content previously.

**Comment by e0i** [comment]
@nenonix Minor style checks needed for this PR. Thanks.

File: windows/deployment/update/waas-delivery-optimization-reference.md [hunk 1]
```diff
@@ -81,8 +81,8 @@ Additional options available that control the impact Delivery Optimization has o
...
 - [Monthly Upload Data Cap](#monthly-upload-data-cap) controls the amount of data a client can upload to peers each month.
 - [Minimum Background QoS](#minimum-background-qos) lets administrators guarantee a minimum download speed for Windows updates. This is achieved by adjusting the amount of data downloaded directly from Windows Update or WSUS servers, rather than other peers in the network.
-- [Maximum Foreground Download Bandwidth](#maximum-foreground-download-bandwidth) specifies the maximum foreground download bandwidth that Delivery Optimization uses across all concurrent download activities as a percentage of available download bandwidth.
-- [Maximum Background Download Bandwidth](#maximum-background-download-bandwidth) specifies the maximum background download bandwidth that Delivery Optimization uses across all concurrent download activities as a percentage of available download bandwidth.
+- [Maximum Foreground Download Bandwidth](#maximum-foreground-download-bandwidth) specifies the **maximum foreground download bandwidth** that Delivery Optimization uses across all concurrent download activities as a percentage of available download bandwidth.
```
Comment (nenonix) [inline windows/deployment/update/waas-delivery-optimization-reference.md]:
```suggestion
- [Maximum Foreground Download Bandwidth](#maximum-foreground-download-bandwidth) specifies the **maximum foreground download bandwidth** that Delivery Optimization uses, across all concurrent download activities, as a percentage of available download bandwidth.
```

File: windows/deployment/update/waas-delivery-optimization-reference.md [hunk 2]
```diff
@@ -81,8 +81,8 @@ Additional options available that control the impact Delivery Optimization has o
...
+- [Maximum Background Download Bandwidth](#maximum-background-download-bandwidth) specifies the **maximum background download bandwidth** that Delivery Optimization uses across all concurrent download activities as a percentage of available download bandwidth.
```
Comment (nenonix) [inline windows/deployment/update/waas-delivery-optimization-reference.md]:
```suggestion
- [Maximum Background Download Bandwidth](#maximum-background-download-bandwidth) specifies the **maximum background download bandwidth** that Delivery Optimization uses, across all concurrent download activities, as a percentage of available download bandwidth.
```

**Comment by j0rt3g4** [comment]
@nenonix  it's done :) thank you!

**Comment by e0i** [comment]
@jaimeo Would you review and merge this PR that addresses minor formatting issues? Thank you.

**Comment by jaimeo** [comment]
Those commas are not necessary and really shouldn't be there.

**Pull request merged**